### PR TITLE
Fixes from probe controller refactor

### PR DESCRIPTION
**Pull request by boks1971**

### CODE REVIEW

File: pkg/service/docker_test.go [hunk 1]
```diff
@@ -19,9 +19,10 @@ import (
 	"log"
 	"net"
 	"os"
-	"sync/atomic"
 	"testing"
 
+	"go.uber.org/atomic"
```
Comment (boks1971) [inline pkg/service/docker_test.go:24]: drive-by to consistently use uber/atomic

**Review by paulwe — APPROVED** [review]
